frontend/src/scenes/insights/InsightsTable/index.tsx

```diff
@@ -1,135 +1,2 @@
-    const { cohorts } = useValues(cohortsModel)
-    const isSingleEntity = indexedResults.length === 1
-
-    if (indexedResults.length === 0 || !indexedResults?.[0]?.data) {
-        return null
-    }
-
-    // Build up columns to include. Order matters.
-    const columns: ColumnsType<IndexedTrendResult> = []
-
-    if (isLegend) {
-        columns.push({
-            title: '',
-            render: function RenderCheckbox({}, item: IndexedTrendResult, index: number) {
-                // legend will always be on insight page where the background is white
-                return (
-                    <PHCheckbox
-                        color={getChartColors('white')[index]}
-                        checked={visibilityMap[item.id]}
-                        onChange={() => toggleVisibility(item.id)}
-                        disabled={isSingleEntity}
-                    />
-                )
-            },
-            fixed: 'left',
-            width: 60,
-        })
-    }
-
-    columns.push({
-        title: 'Label',
-        render: function RenderLabel({}, item: IndexedTrendResult) {
-            return (
-                <span
-                    style={{ cursor: isSingleEntity ? undefined : 'pointer' }}
-                    onClick={() => !isSingleEntity && toggleVisibility(item.id)}
-                >
-                    {formatLabel(item)}
-                </span>
-            )
-        },
-        fixed: 'left',
-        width: 150,
-    })
-
-    if (showTotalCount) {
-        columns.push({
-            title: 'Total',
-            dataIndex: 'count',
-            fixed: 'left',
-            width: 100,
-        })
-    }
-
-    if (filters.breakdown) {
-        columns.push({
-            title: 'Breakdown Value',
-            render: function RenderBreakdownValue({}, item: IndexedTrendResult) {
-                return formatBreakdownLabel(item.breakdown_value, cohorts)
-            },
-            fixed: 'left',
-            width: 150,
-        })
-    }
-
-    if (indexedResults && indexedResults.length > 0) {
-        const valueColumns = indexedResults[0].data.map(({}, index: number) => ({
-            title: (indexedResults[0].labels || indexedResults[0].dates || indexedResults[0].days)[index],
-            render: function RenderPeriod({}, item: IndexedTrendResult) {
-                return maybeAddCommasToInteger(item.data[index])
-            },
-        }))
-
-        columns.push(...valueColumns)
-    }
-
-    return (
-        <Table
-            dataSource={indexedResults}
-            columns={columns}
-            size="small"
-            rowKey="id"
-            pagination={{ pageSize: 100, hideOnSinglePage: true }}
-            style={{ marginTop: '1rem' }}
-            scroll={indexedResults && indexedResults.length > 0 ? { x: indexedResults[0].data.length * 160 } : {}}
-            data-attr="insights-table-graph"
-        />
-    )
-}
+import { InsightsTable } from './InsightsTable'
```

Comment:
The most minor nit: I think this can be `export { InsightsTable } from './InsightsTable'`